.prettierignore

```diff
@@ -0,0 +1,4 @@
+node_modules/
+package-lock.json
+yarn.lock
+package.json
```

Comment:
I think formatting of package.json is a good idea

Reply:
It's good idea